package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chainlink",
-  "version": "2.15.0-mercury",
+  "version": "2.15.0",
```

Comment:
this causes integration tests to fail - need to check with RELENG after merging this as to what it should be when cutting an "official" release